rewatch/tests/suite-ci.sh

```diff
@@ -18,8 +18,11 @@ export RESCRIPT_BSC_EXE
 
 source ./utils.sh
 
+bold "Yarn install"
+(cd ../testrepo && yarn)
+
 bold "Rescript version"
-(cd ../testrepo && ./node_modules/.bin/rescript -v)
+(cd ../testrepo && ./node_modules/.bin/rescript --version)
```

Comment:
`-v` is verbosity and this was trying to compile the testrepo with an older installed version of rescript.
I'm not sure how this wasn't leading to bigger problems.